src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -723,4 +767,365 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 		}
 
 	}
+
+	//my code here:
+	private createNewTable(): void {
+
+		let t: Table<any>;
+		if (this.placeHolderDataSets) {
+			let dataSet = this.placeHolderDataSets[0];
+			let options = {
+				enableCellNavigation: true,
+				enableColumnReorder: false,
+				renderRowWithRange: true,
+				showHeader: true,
+				rowHeight: this._rowHeight,
+				defaultColumnWidth: 120,
+				editable: false,
+				autoEdit: false,
+				enableAddRow: false, // TODO change when we support enableAddRow
+				enableAsyncPostRender: false,
+				editorFactory: {
+					getEditor: (column: ISlickColumn<any>) => this.getColumnEditor(column)
+				},
+				formatterFactory: {
+					getFormatter: this.getFormatter
+				}
+			};
+
+			if (dataSet.columnDefinitions) {
+				t = new Table(this.nativeElement, { dataProvider: new AsyncDataProvider(dataSet.dataRows), columns: dataSet.columnDefinitions }, options);
+
+				this._tables[0] = t;
+				if (this.selectionModel) {
+					if (typeof this.selectionModel === 'object') {
+						t.grid.setSelectionModel(this.selectionModel);
+					} else if (typeof this.selectionModel === 'string' && Slick[this.selectionModel] && typeof Slick[this.selectionModel] === 'function') {
+						t.grid.setSelectionModel(Slick[this.selectionModel]);
+					} else {
+						console.error(`Tried to register selection model ${this.selectionModel},
+						but none was found to be attached to Slick Grid or it was not a function.
+						Please extend the Slick namespace with the selection model as a function before registering`);
+					}
+				}
+				for (let plugin of this.plugins[0]) {
+					this._tables[0].registerPlugin(plugin);
+				}
+
+				this._columnNameToIndex = {};
+				for (let i = 0; i < this.placeHolderDataSets[0].columnDefinitions.length; i++) {
+					this._columnNameToIndex[this.placeHolderDataSets[0].columnDefinitions[i].name] = i;
+				}
+			}
+		}
+		else {
+			this._tables[0] = new Table(this.nativeElement);
+		}
+	}
+
```

Comment:
nit: extra blank lines.  Please try to be aware of document formatting conventions when adding code (e.g. 1 blank line between related methods usually, etc.).